ingestion/functions/parsing/brazil_paraiba/paraiba.py

```diff
@@ -0,0 +1,191 @@
+    elif "teste" in raw_test.lower():
+        #this is reported as both Teste Rapido and teste rapido
+        return "Serological test"  
+    else:
+        print(f'unknown confirmation method: {raw_test}')
+        return "Unknown"
+
+def convert_preexisting_conditions(raw_commorbidities: str):
+    preexistingConditions = {}
+    if raw_commorbidities not in ["Sem comorbidades","Doença Hematológica","Tabagismo","Imunossupressão"]:
+        preexistingConditions["hasPreexistingConditions"] = True
+        commorbidities = {
+                        "Diabetes Mellitus": "diabetes mellitus", 
+                        "Cardiopatia": "heart disease",
+                        "Doença de Aparelho Digestivo": "gastrointestinal system disease",
+                        "Doença Hepática": "liver disease",
+                        "Doença Neurológica": "nervous system disease",
+                        "Doença Renal": "kidney disease",
+                        "Doença Respiratória": "respiratory system disease",
+                        "Neoplasia": "neoplasm",
+                        "Etilismo": "alcohol use disorder",
+                        "Obesidade": "obesity",
+                        "Transtorno mental": "mental or behavioural disorder",
+                        "Hipertensão": "hypertension"
+                        }
+        
+        commorbidities_list = []
+
+        for key in commorbidities:
+            if key in raw_commorbidities:
+                commorbidities_list.append(commorbidities[key])
+
+        preexistingConditions["values"] = commorbidities_list
+        return preexistingConditions
+    else:
+        return None         
+
+def convert_location(raw_entry: str):
+    query = ", ".join(word for word in [raw_entry, "Paraíba", "Brazil"] if word)
+    return {"query": query}
+
+def convert_notes(raw_commorbidities: str):
+    raw_notes = []
+    if "Imunossupressão" in raw_commorbidities:
+        raw_notes.append("Patient with immunosupression")
+    if "Tabagismo" in raw_commorbidities:
+        raw_notes.append("Smoker")
+    if "Doença Hematológica" in raw_commorbidities: 
+    	raw_notes.append("Hematologic disease")
+    if "Outros" in raw_commorbidities: 
+    	raw_notes.append("Unspecified pre-existing condition")
+    notes = (', ').join(raw_notes)
+    return notes
+
+
+def parse_cases(raw_data_file: str, source_id: str, source_url: str):
+    """Parses G.h-format case data from raw API data.
+        Caveats:
+        1. There are no patient ID/case ID in the raw API so we aren't able
+           to dedupe.
+        2. This data only includes deaths from Covid-19 so the outcome of all cases will be death.
+        3. There is no date of confirmation.
+    """
+    with open(raw_data_file, "r") as f:
+        reader = csv.reader(f)
+        next(reader) # Skip the header.
+        cases = []
+        for row in reader:
+            if row[_DATE_SYMPTOMS_INDEX] < "2019-11-01": #One date is recorded as year 2000
```

Comment:
you need to parse dates to do this, this will do a string comparison which will use length and not the actual values in the string.